neptune/new/exceptions.py

```diff
@@ -492,7 +492,7 @@ def __init__(self):
 
 class NeptuneConnectionLostException(NeptuneException):
     def __init__(self):
-        message = """
+        message = """https://docs.neptune.ai/administration/workspace-project-and-user-management/projects
```

Comment:
Something wrong went here I think. There should be anything before the exception header.